features/mbedtls/platform/COMPONENT_PSA_SRV_IMPL/inc/default_random_seed.h

```diff
@@ -8,7 +8,7 @@ extern "C" {
 #endif
 
 #ifndef RANDOM_SEED_ITS_UID
-    #define MBED_RANDOM_SEED_ITS_UID 0x90210
+#define MBED_RANDOM_SEED_ITS_UID 0x90210
```

Comment:
Could we get this define from the Mbed Crypto library instead?

This still doesn't look very random...

Reply:
This feature does not enforce the use of Mbed Crypto,
There are 2 things the injection and the get/set the seed from secure storage

this definition is relevant for this file...

Reply:
replacing number using this tool https://www.random.org/bytes/

Reply:
If this feature does not require the use of Mbed Crypto, how does the initial random seed get populated? If there is some way other than Mbed Crypto, then we should not worry about making any PSA Entropy Injection API since we can just use that other way, right?

Reply:
@Patater these 2 api callbacks currently are in no way related to any library except psa_its* secure storage, this is the only link. But psa-crypto is not part of the includes. let's not make a circular dependency where psa-crypto uses callbacks and callbacks include psa crypto

Reply:
I'm suggesting we make Mbed Crypto the one source of truth for which storage slot contains the NV seed. Both have to agree. It's backwards to make Mbed Crypto depend on Mbed OS to determine this. Shouldn't be circular, but from Mbed Crypto picking the slot and Mbed OS using the same slot.